app/routables/components/modal/template.hbs

```diff
@@ -0,0 +1,17 @@
+<!-- <div id="modal">
```

Comment:
Are we not using this modal component at all?

Reply:
I think I will use a modal on the contact section of the menu. Short concise and effective.